pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for the front-end web development.
+> A package can be a GitHub shorthand, a Git endpoint, a URL, a registered package.
```

Comment:
I think I could change the first command example to reflect all the possible cases like so:

```
bower install {{package_name}} {{endpoint}} {{user/package}} {{url}}
```

But then, it almost more confusing. I think I should just remove the "Github shorthand" part since it adds little.